CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 10.1.5
```

Comment:
Desktop access playback speeds should be in this too.

Reply:
They were in 10.1.4?

Reply:
🤦‍♂️